Tests/AppTests/Controllers/User/UserControllerTests.swift

```diff
@@ -0,0 +1,187 @@
+import XCTest
+import Testing
+import HTTP
+import Sockets
+@testable import Vapor
+@testable import App
+
+//swiftlint:disable superfluous_disable_command
+//swiftlint:disable force_try
+class UserControllerTests: TestCase {
+  var drop: Droplet!
+  
+  let firstName = "Cocoa"
+  let lastName = "Heads"
+  let company = "CocoaHeads"
+  let position = "Tester"
+  let email = "tester@cocoaheads.ru"
+  let phone = "1234567890"
+  let photo = "http://cocoaheads.ru/photo.jpg"
+  let updatedLastName = "Feet"
+
+  override func setUp() {
+    super.setUp()
+    do {
+      drop = try! Droplet.testable()
+    } catch {
+      XCTFail("Droplet set raise exception: \(error.localizedDescription)")
+      return
+    }
+  }
+
+  func testThatUserCreatesFromRequest() throws {
```

Comment:
`testThatUserIsCreatedFromRequest` is more correct i think

Reply:
Fixed